src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -114,7 +114,7 @@ private function validate($content, $file = null)
         try {
             $this->getParser()->parse($content, Yaml::PARSE_CONSTANT);
         } catch (ParseException $e) {
-            return array('file' => $file, 'valid' => false, 'message' => $e->getMessage());
+            return array('file' => $file, 'line' => $e->getParsedLine(), 'valid' => false, 'message' => $e->getMessage());
```

Comment:
I've added `getLastLineNumberBeforeDeprecation` to get the last line read before the deprecation was triggered. Do you think this is the way to go? The only other way I see to get line numbers from these `trigger_error` is by adding them into the `$error_msg` and parse them out.. But this seems clearer even though not perfect (it's a very specific method). Maybe you think we should make `getRealCurrentLineNb` public?